packages/waku/src/router/client.ts

```diff
@@ -301,8 +305,15 @@ const InnerRouter = ({ routerData }: { routerData: RouterData }) => {
   const refetch = useRefetch();
 
   const [route, setRoute] = useState(() =>
-    parseRoute(new URL(window.location.href)),
+    parseRoute(new URL(window.location.href), true),
```

Comment:
So, `ignoreHash` is used only here.

Reply:
Yes, without this, there are hydration errors, because the server has no hash, but the first client render does.

Reply:
Oh, I see. Can you leave some comments in the code?
I might refactor things around here in the future.

Reply:
I added some explanations there